atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/pue/ResilientCommitTimestampPutUnlessExistsTableTest.java

```diff
@@ -82,6 +86,52 @@ public void pueThatThrowsIsCorrectedOnGet() throws ExecutionException, Interrupt
         verify(spiedStore).put(anyMap());
     }
 
+    @Test
+    public void onceNonNullValueIsReturnedItIsAlwaysReturned() {
+        PutUnlessExistsTable<Long, Long> putUnlessExistsTable = new ResilientCommitTimestampPutUnlessExistsTable(
+                new CassandraImitatingConsensusForgettingStore(0.5d), TwoPhaseEncodingStrategy.INSTANCE);
+
+        for (long startTs = 1L; startTs < 1000; startTs++) {
```

Comment:
nit: probably want `<=` (unless it's intentional that the core loop runs 999 times).

Reply:
There is no rational reason why exactly 1000 iterations is better than 999 :P